src/main/java/thePackmaster/ThePackmaster.java

```diff
@@ -130,8 +130,7 @@ public String getLocalizedCharacterName() {
 
     @Override
     public AbstractCard getStartCardForEvent() {
-        System.out.println("YOU NEED TO SET getStartCardForEvent() in your " + getClass().getSimpleName() + " file!");
-        return null;
+        return new Rummage();
```

Comment:
I picked the attack rather than the skill. The base game does the same for every character other than Defect, who doesn't have a starter attack (Bash for Ironclad, Neutralize for Silent, Eruption for Watcher). To be fair, Ironclad doesn't have a starter skill, so the precedent is really just Silent and Watcher.